output.go

```diff
@@ -0,0 +1,102 @@
+package plugin
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Output is the JSON payload that gets sent to Stderr from the plugin to the host
+type Output struct {
+	Message   string    `json:"message"`
+	Level     string    `json:"level"`
```

Comment:
Do we want to provide preexisting levels that match hclog's levels for ease of use? Even if not technically less typing, simply providing them will give a hint to the user as to which levels are supported in a built-in fashion.